web/cookiesCTF/main.py

```diff
@@ -0,0 +1,122 @@
+from bottle import route, run, request, response, static_file, get, template
+
+
+@route('/')
+def hello():
+	f = open('./Pages/index.html')
+	return f.read()
+
+
+@get('/styles.css')
+def retCss():
+	return static_file('styles.css', root='./Pages')
+
+
+@get('/', method='POST')
+def checkFlag():
+	flag = request.forms.get('flag')
+	if flag == "PB{eat_my_cookies}" or flag == "PB{I'm_a_cookiebot}" or flag == "PB{Jake_don't_mind}" or flag == "PB{Jake_isn't_lonely}" or flag == 'PB{Y0tub3_v1d30s_ar3_fun}':
+		f = open('./Pages/Correct.html')
+		return f.read()
+	else:
+		f = open('./Pages/TryAgain.html')
+		return f.read()
+
+
+@route('/1')
+def task1():
+	response.set_cookie("is_admin", "no", path="/")
+	if request.get_cookie("is_admin") == "no":
+		f = open('./Pages/Task1/Task1GUEST.html')
+		return f.read()
+	elif request.get_cookie("is_admin") == "yes":
+		f = open('./Pages/Task1/Task1ADMIN.html')
+		return f.read()
+
+
+@route('/2')
+def task2():
+	response.set_cookie("is_robot", "99e9bae675b12967251c175696f00a70", path="/")
+	if request.get_cookie("is_robot") == "87b7cb79481f317bde90c116cf36084b":
+		f = open('./Pages/Task2/Task2ROBOT.html')
+		return f.read()
+	else:
+		f = open('./Pages/Task2/Task2HUMAN.html')
+		return f.read()
+
+@route('/3')
+def task3():
+	response.set_cookie("Jake_Surname", "Jake-None", path="/")
+	if request.get_cookie("Jake_Surname") == "Jake-Cake":
+		f = open('./Pages/Task3/Task3SOLVED.html')
+		return f.read()
+	else:
+		f = open('./Pages/Task3/Task3notSOLVED.html')
+		return f.read()
+
+
+@route('/3_file1')
+def task3_file1():
+	return static_file('encoded_string.txt', root='./')
+
+
+@route('/3_file2')
+def task3_file2():
+	return static_file('ciphers.png', root='./', mimetype='image/png')
+
+
+@route('/4')
+def task4():
+	response.set_cookie("Jake_is_fun",
+						"35eae3ea7fe1598f7f8ffa2806229068074bac921b2d235ed02660c6009eba9d40b09e2f5bfa099c8e66235d89d2bd3771941741a02a181aba9dcb027acca465",
+						path='/')
```

Comment:
That's strange formatting. I don't mind having very long lines, because it's a CTF task source, not a production code or something, but this one's really strange